src/Raven.Server/Documents/Sharding/ShardLocator.cs

```diff
@@ -5,6 +5,25 @@ namespace Raven.Server.Documents.Sharding
 {
     public static class ShardLocator
     {
+        public static Dictionary<int, List<string>> GroupIdsByShardIndex(IEnumerable<string> ids,
```

Comment:
Maybe introduce a dedicated class for the result instead of the dict?

Reply:
done

Reply:
Ping